modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc

```diff
@@ -151,9 +140,34 @@ Alternatively you may install [systemitem]``mgrpxy-zsh-completion`` or [systemit
 
 . Reboot the system.
 
+== Deploy {productname} Proxy on {sles} {bci-suma}
+
+Alternatively, you can install {productname} Proxy on {sles} {bci-suma}.
+The following procedure describes the main steps of the installation process.
+
+.Procedure: Installing {productname} Proxy on {sles} {bci-suma}
+. Locate and download {sles} {bci-suma} [literal]``.iso`` at https://www.suse.com/download/sles/.
+. Make sure that the {scc} code you plan to use has the Extensions enabled.
+. Start the installation of {sles} {bci-suma}.
+  .. On the [literal]``Language, keyboard and product selection`` select the product to install.
+  .. On the [literal]``License agreement`` read the agreement and check [guimenu]``I Agree to the License Terms``.
```

Comment:
I do not know and cannot test ATM whether the indentation will cause strange rendering or not.  I usually do not indent substeps (`..`).